qstash/api/api-ratelimiting.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: "API Rate Limit Response"
+description: "This page documents the rate limiting behavior of our API and explains how to handle different types of rate limit errors."
+---
+
+## Overview
+
+Our API implements rate limiting to ensure fair usage and maintain service stability. There are three types of rate limits:
+
+1. **Daily rate limit**
+2. **Burst rate limit**
+3. **Chat-based rate limit**
+
+When a rate limit is exceeded, the API returns a 429 status code along with specific headers that provide information about the limit, remaining requests/tokens, and reset time.
+
+### Daily Rate Limit
+
+This is a **daily** limit applied across all API endpoints.
+
+**Headers**:
+
+- `RateLimit-Limit`: Maximum number of requests allowed per day
+- `RateLimit-Remaining`: Remaining number of requests for the day
+- `RateLimit-Reset`: Time (in unix timestamp) when the daily limit will reset 
+
+### Burst Rate Limit
+
+This is a short-term limit **per second** to prevent rapid bursts of requests.
```

Comment:
Since the daily limit applies only to publish-related requests, we can emphasize that the burst limit applies to **_all_** API endpoints.